src/gui/MidiClipView.cpp

```diff
@@ -140,8 +142,37 @@ void MidiClipView::changeName()
 
 
 
+void MidiClipView::transposeSelection()
+{
+	int semitones = QInputDialog::getInt(this, tr("Transpose"), tr("Semitones to transpose:"), 0, -NumKeys, NumKeys, 1);
+	if (semitones == 0) { return; }
+
+	// TODO make this not crash
+	// Engine::getSong()->addJournalCheckPoint();
+
+	QSet<Track*> m_changedTracks;
+	for (ClipView* clipview: getClickedClips())
```

Comment:
QtCreator smashes a [warning](https://github.com/KDE/clazy/blob/1.11/docs/checks/README-range-loop-detach.md) here.

Reply:
Actually I think this warning can not be fixed. I can't make the `clipview` `const`... Does that warning make any sense?